models/google_play__device_report.sql

```diff
@@ -39,6 +39,7 @@ with installs as (
         -- gonna need to do some first_value stuff here...
         install_metrics.total_unique_user_installs,
         install_metrics.total_device_installs,
+        install_metrics.total_device_uninstalls,
```

Comment:
Could you elaborate on the first_value stuff? You didn't take this approach in the country report. Is the comment just a holdover from before?

Reply:
ah i meant using the `first_value()` window function, which i do use in the country report as well. changing the comment anyways to refer to this as backfilling